osdk/src/cli.rs

```diff
@@ -6,8 +6,8 @@ use clap::{crate_version, Args, Parser};
 
 use crate::{
     commands::{
-        execute_build_command, execute_check_command, execute_clippy_command, execute_new_command,
-        execute_run_command, execute_test_command,
+        execute_build_command, execute_new_command, execute_run_command, execute_test_command,
+        execute_wrapped_command,
```

Comment:
The wrapped command sounds weird to me. How about forwarded commands or proxied commands or redirected commands?